meta-luneos/recipes-support/sensorfw/sensorfw.bb

```diff
@@ -40,12 +40,14 @@ EXTRA_QMAKEVARS_PRE += "MAKE_DOCS=no "
 EXTRA_QMAKEVARS_PRE_append_halium = "CONFIG+=autohybris "
 
 # Halium-9.0 devices use binder to communicate with sensors
-EXTRA_QMAKEVARS_PRE_append_tissot = "CONFIG+=binder "
-EXTRA_QMAKEVARS_PRE_append_rosy = "CONFIG+=binder "
+EXTRA_QMAKEVARS_PRE_append_hammerhead = "CONFIG+=binder "
+EXTRA_QMAKEVARS_PRE_append_mako = "CONFIG+=binder "
 EXTRA_QMAKEVARS_PRE_append_mido = "CONFIG+=binder "
-EXTRA_QMAKEVARS_PRE_append_yggdrasil = "CONFIG+=binder "
+EXTRA_QMAKEVARS_PRE_append_rosy = "CONFIG+=binder "
 EXTRA_QMAKEVARS_PRE_append_sagit = "CONFIG+=binder "
-EXTRA_QMAKEVARS_PRE_append_hammerhead = "CONFIG+=binder "
+EXTRA_QMAKEVARS_PRE_append_tissot = "CONFIG+=binder "
+EXTRA_QMAKEVARS_PRE_append_yggdrasil = "CONFIG+=binder "
```

Comment:
at this point, we could integrate it in the append_halium

Reply:
@tofee well we still have some unofficial targets on 7.1, so I guess better to keep it like this for now, unless we can do halium_9.0 somehow ;)

Reply:
Right